pkg/resource/promtail-toggle/observability_bundle_configmap.go

```diff
@@ -25,7 +25,7 @@ func ObservabilityBundleConfigMapMeta(lc loggedcluster.Interface) metav1.ObjectM
 		Namespace: lc.GetAppsNamespace(),
 		Labels: map[string]string{
 			// This label is used by cluster-operator to find extraconfig
-			"app.kubernetes.io/name": lc.AppConfigName("observability-bundle"),
+			"app.kubernetes.io/name": "observability-bundle",
```

Comment:
That's true but not that is will only work on vintage WCs, Could you mention it in the comment?

Reply:
Good point !

Reply:
This is why we were using a function from the cluster type-specific package, so we return a different value depending on whether we're on CAPI or vintage.

Reply:
Oh then I guess this implementation needs to be fixed

Reply:
I see this comment as "resolved", but yet we still have a hardcoded value that will not work in all situations.

Reply:
Oh I was too eager Yesterday

Reply:
I am currently reworking on it